modules/virt-checking-storage-class.adoc

```diff
@@ -13,7 +13,7 @@ Cinder, the default storage class, does not support VM import. See link:https://
 
 You can check the default storage class in the {product-title} console. If the default storage class is not NFS, you can change the default storage class so that it is no longer the default and change the NFS storage class so that it is the default.
 
-If more than one default storage class is defined, the VirtualMachineImport CR uses the default storage class that is first in alphabetical order.
+If more than one default storage class is defined, the virtual machine import CR uses the default storage class that is first in alphabetical order.
```

Comment:
Not sure about this one

Reply:
I think if you're saying "custom resource," it's like saying "object," so it should be `VirtualMachineImport`